docs: add self-hosted audio models guide with Voxtral Small/vLLM (#898)
agentgateway/website: content/docs/kubernetes/latest/llm/providers/audio.md

---

> title: Audio models (Whisper, Voxtral)

Reviewer: Is Whisper available through Voxtral or something? I didn't notice it mentioned in the guide.
```suggestion
title: Audio models
```

weight: 60

> description: Configure audio models with Passthrough routing for transcription and speech endpoints.

Reviewer: Was there a speech endpoint? maybe just transcription
```suggestion
description: Configure self-hosted audio models such as Voxtral with passthrough routing for transcription endpoints.
```
